### PR TITLE
Update libwebp to 1.0.3, fixing CVEs.

### PR DESCRIPTION
**Pull request by MateuszMalisz**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update libwebp to 1.0.3 to fix various CVEs.

###### Change Log  

- Update libwebp to 1.0.3

###### Does this affect the toolchain?  

NO

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2018-25009
- https://nvd.nist.gov/vuln/detail/CVE-2018-25010
- https://nvd.nist.gov/vuln/detail/CVE-2018-25011
- https://nvd.nist.gov/vuln/detail/CVE-2018-25012
- https://nvd.nist.gov/vuln/detail/CVE-2018-25013
- https://nvd.nist.gov/vuln/detail/CVE-2018-25014
- https://nvd.nist.gov/vuln/detail/CVE-2020-36329
- https://nvd.nist.gov/vuln/detail/CVE-2020-36330
- https://nvd.nist.gov/vuln/detail/CVE-2020-36331

###### Test Methodology

- Local build

### CODE REVIEW

File: SPECS/libwebp/libwebp.spec [hunk 1]
```diff
@@ -64,9 +64,10 @@ find %{buildroot} -name '*.la' -delete
 %{_libdir}/pkgconfig/*.pc
 
 %changelog
-* Sat May 09 00:21:21 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.0.0-4
-- Added %%license line automatically
-
+*   Tue May 25 2021 Mateusz Malisz <mamalisz@microsoft.com> 1.0.3-1
+-   Update to version 1.0.3.
```
Comment (jslobodzian) [inline SPECS/libwebp/libwebp.spec:68]: nit: add spaces between changelog entries...because linter.

File: SPECS/libwebp/libwebp.signatures.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libwebp-1.0.0.tar.gz": "c5c5ebf979543ac1f3348df8f6245262abd787a147b9632c880d92bfc38dbbeb"
+     "libwebp-1.0.3.tar.gz": "082d114bcb18a0e2aafc3148d43367c39304f86bf18ba0b2e766447e111a4a91"
```
Comment (PawelWMS) [inline SPECS/libwebp/libwebp.signatures.json:3]: Nit: additional whitespaces.

File: SPECS/libwebp/libwebp.signatures.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
  }
-}
```
Comment (PawelWMS) [inline SPECS/libwebp/libwebp.signatures.json:5]: Nit: unnecessary new line - I think we don't have this in other/most signature JSONs. I don't think it makes any difference for our needs, though.

File: SPECS/libwebp/libwebp.spec [hunk 4]
```diff
@@ -64,9 +64,10 @@ find %{buildroot} -name '*.la' -delete
...
+*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> 1.0.0-4
```
Comment (PawelWMS) [inline SPECS/libwebp/libwebp.spec:68]: Thanks for fixing the changelog!

**Review by PawelWMS — APPROVED** [review]

File: SPECS/libwebp/libwebp.signatures.json (hunk 3 above)
Comment (MateuszMalisz) [reply SPECS/libwebp/libwebp.signatures.json:5]: If I am not mistaken, files without the newline at the end are not POSIX compliant: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

**Pull request merged**